src/runtimeConfig.ts

```diff
@@ -29,10 +29,10 @@ export interface RuntimeConfig {
 	notificationWebhookUrl: string;
 	torrents: string[];
 	port: number | null;
-	searchCadence: number;
-	rssCadence: number;
-	snatchTimeout: number;
-	searchTimeout: number;
+	searchCadence?: number;
+	rssCadence?: number;
+	snatchTimeout?: number;
+	searchTimeout?: number;
```

Comment:
these are nullish

Reply:
I'm not sure I understand what you're getting at, in the Zod schema, they are listed as nullish() - and given that they are optional here, it seems I've already got this covered properly?

Reply:
Did some testing, with both null and undefined it was previously fine, this does not get translated from fileConfig -> runtimeConfig at all if it's undefined, and null gets stored as null.

I'm not sure if you were expecting issues with this, but I don't believe anything was wrong?

Reply:
Not at runtime, but the types were wrong. Without it  marked as optional, we get bogus IDE messages like this, and it also means there's a possibility of introducing a bug later by forgetting to check for null/undefined.

<img width="909" alt="image" src="https://github.com/cross-seed/cross-seed/assets/9119498/38a03c43-4c39-41ba-aeab-687423553fd4">